web/index.php

```diff
@@ -55,8 +62,16 @@
     $twilio->say(sprintf("The current on-call engineer is %s." .
         "%s Please hold while we connect you.",
         $user['first_name'], $time), $attributes);
-
-    $dial = $twilio->dial(NULL, array('action' => "check_if_completed_by_human.php", 'timeout' => 25));
+    
+    $dialvars = array('action' => "check_if_completed_by_human.php", 'timeout' => 25);
+    if ($record != strtolower("false")) {
+        if ($record == strtolower("true")) {
```

Comment:
I made some small changes here because this didn't seem to do what I think it was expected to do. I used the following instead:

`if (strtolower($record) == 'true') {`

I made a similar change to the if statement above.